### PR TITLE
Sprint_5_Deliverables V2

### PR DESCRIPTION
**Pull request by Jyyunrin**

### CODE REVIEW

**Review by LouiseYLi — COMMENTED** [review]
LGTM

**Review by LouiseYLi — APPROVED** [review]
LGTM

File: src/controllers/collection.js [hunk 1]
```diff
@@ -25,16 +25,16 @@ const collectionSearchPOST = async (req, res) => {
     // Database read with conditional sorting based off of selected sort option from dropdown options
     switch (sort) {
     case 'alpha':
-        collections = collectionService.sortAlpha(userId, regexPattern)
+        collections = await collectionService.sortAlpha(userId, regexPattern)
```
Comment (wyliao76) [inline src/controllers/collection.js:28]: In the future refactor this switch case into a function with a return of collections.

**Review by wyliao76 — APPROVED** [review]
Please refactor based on functional programming and add unit tests in the future.

**Pull request merged**